config/scripts/optoutauthselect.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Tell build process to exit if there are any errors.
+set -oue pipefail
+
+echo "Opt-out of 'authselect' profile generation"
```

Comment:
Can you clarify what this does? Is authselect profile generation done by default?

Reply:
Authselect is an administration tool red hat distributions use to manage pam services. The service configurations in /etc/pam.d are generated by this tool. Wîth the opt-out command we disable this functionality and ensure users don't overwrite the pam service script.